package.json

```diff
@@ -16,6 +16,8 @@
     "startCommand": "yarn && yarn dev"
   },
   "devDependencies": {
-    "concurrently": "^8.1.0"
+    "concurrently": "^8.1.0",
```

Comment:
Any reason why this packages got added? WIth yarn workspace this should not have happened :o

Reply:
I set this project in a vanilla environment up. I installed node, npm and yarn and tried running `npm run dev:server`, where i received the error, that the command `tsx`was not available. I added ts-node and typescript through npm install, which possibly added this one too. I just double checked and "concurrently" seems to be not required. Ill fix the PR.